src/ledger.test.ts

```diff
@@ -389,12 +390,12 @@ describe("ledger", () => {
     function interactionBuilder(
       policyHmac?: string,
       verify?: boolean,
-      name = "satoshi's wallet",
-      braid = TEST_FIXTURES.braids[0]
+      walletConfig = braidDetailsToWalletConfig(
+        TEST_FIXTURES.braids[0] as unknown as BraidDetails
```

Comment:
The double `as` always throws me off, maybe just a preference thing, but I always preferred `<unknown>TEST_FIXTURES.braids[0] as BraidDetails`. If anything, it's a couple characters shorter. I just read it in my head as "Unknown test fixture as BraidDetails".

Reply:
turns out wasnt' needed at all, but this is a cool pattern. 
Removed: 6035a7c